src/server/editorServices.ts

```diff
@@ -588,11 +588,16 @@ namespace ts.server {
             return this.findExternalProjectByProjectName(projectName) || this.findConfiguredProjectByProjectName(toNormalizedPath(projectName));
         }
 
-        getDefaultProjectForFile(fileName: NormalizedPath, refreshInferredProjects: boolean) {
-            if (refreshInferredProjects) {
+        getDefaultProjectForFile(fileName: NormalizedPath, ensureProject: boolean) {

```

Comment:
This is a completely public method. It's not a breaking change at the type level, but it looks like you're changing the boolean parameter to have a different meaning?

Reply:
I understand but earlier derfinition wasnt anyways quite clear on what it should do and shouldnt do. This seems better